### PR TITLE
test: fix, refactor, cleanup

### PR DESCRIPTION
**Pull request by kevinbheda**

### CODE REVIEW

**Comment by ravisuhag** [comment]
@kevinbheda Why a PR with just TODOs. I would suggest adding these things and then raise a PR.

**Comment by kevinbheda** [comment]
yes the plan is that :)

File: dagger-core/src/test/java/io/odpf/dagger/core/processors/longbow/data/LongbowTableDataTest.java [hunk 1]
```diff
@@ -66,20 +66,10 @@ public void shouldReturnMultipleListOfStringWhenLongbowDataMoreThanOne() {
         LongbowSchema longbowSchema = new LongbowSchema(columnNames);
         LongbowTableData longbowTableData = new LongbowTableData(longbowSchema);
         Map<String, List<String>> actualData = longbowTableData.parse(scanResult);
-        Assert.assertEquals(Arrays.asList("RB-234", "RB-224"), actualData.get("longbow_data1"));
-        Assert.assertEquals(Arrays.asList("RB-235", "RB-225"), actualData.get("longbow_data2"));
-    }
-
-    @Test
-    public void shouldOnlyReturnLongbowDataColumn() {
```
Comment (gauravsinghania) [inline dagger-core/src/test/java/io/odpf/dagger/core/processors/longbow/data/LongbowTableDataTest.java:74]: why did we remove this test?

File: dagger-core/build.gradle [hunk 2]
```diff
@@ -95,6 +95,7 @@ dependencies {
     testImplementation 'org.powermock:powermock-api-mockito2:2.0.0-beta.5'
     testImplementation 'com.google.guava:guava:27.0.1-jre'
     testImplementation 'org.grpcmock:grpcmock-junit5:0.5.0'
+    testImplementation 'com.github.stefanbirkner:system-rules:1.19.0'
 }
```
Comment (gauravsinghania) [inline dagger-core/build.gradle:100]: need to bump version in version.txt file

**Pull request merged**